pkgs/applications/misc/furtherance/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, appstream-glib, desktop-file-utils
+, glib, libadwaita, meson, ninja, pkg-config, python3, wrapGAppsHook4, dbus
+, gtk4, sqlite }:
+stdenv.mkDerivation rec {
+  pname = "Furtherance";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "lakoliu";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "oIzwFPJUY+OwT2B6n7NPg8bMxEd4D0MKPe3GQxrXmEU=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    sha256 = "VlV6EkMKMGoUELUMgEp/OXdTJTifBqsoqrf/eZTyl1k=";
+  };
+
+  nativeBuildInputs = [
+    appstream-glib
+    desktop-file-utils
+    meson
+    ninja
+    pkg-config
+    python3
+    rustPlatform.cargoSetupHook
+    rustPlatform.rust.cargo
+    rustPlatform.rust.rustc
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [ dbus glib gtk4 libadwaita sqlite ];
+
+  # doCheck = true;
```

Comment:
Remove this line if it can't be enabled.

Reply:
Do you mean the # doCheck comment? or the buildInputs? the doCheck can be removed, I was just toying around with that setting.

Reply:
Yes, `doCheck`. Looking at the application, it doesn't seem to have any tests, so you can remove it just fine (unless I'm wrong, please let me know if that's the case).

Reply:
Yeah that line isnt necessary! I should've removed it from the start, sorry about that! pushed the commit, I am incredibly thankful for your patience helping me with all of this! pushed the commit with all your requested changes.